packages/discord.js/src/managers/PollAnswerVoterManager.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const { Collection } = require('@discordjs/collection');
+const { makeURLSearchParams } = require('@discordjs/rest');
+const { Routes } = require('discord-api-types/v10');
+const CachedManager = require('./CachedManager');
+const User = require('../structures/User');
+
+/**
+ * Manages API methods for users who voted on a poll and stores their cache.
+ * @extends {CachedManager}
+ */
+class PollAnswerVoterManager extends CachedManager {
+  constructor(answer, iterable) {
+    super(answer.client, User, iterable);
+
+    /**
+     * The poll answer that this manager belongs to
+     * @type {PollAnswer}
+     */
+    this.answer = answer;
+  }
+
+  /**
+   * The cache of this manager
+   * @type {Collection<Snowflake, User>}
+   * @name PollAnswerVoterManager#cache
+   */
+
+  /**
+   * Options used to fetch voters who voted for this poll answer.
+   * @typedef {Object} FetchPollAnswerVotersOptions
```

Comment:
This definition appears to be the same as `BaseFetchPollAnswerVotersOptions`

Reply:
Looks like I accidentally overrode a type with this one 😅